pixeltable/tests/test_dataframe.py

```diff
@@ -285,7 +285,8 @@ def restrict_json_for_default_collate(obj):
             c_image=t.c_image.resize([220, 224]).convert('RGB')
         )
         df_size = df.count()
-        ds = df.to_pytorch_dataset(image_format='pt')
+        assert df_size >= 23, 'want to test enough tuples'
+        ds = df.to_pytorch_dataset(image_format='pt')        
```

Comment:
since you're explicitly testing the shuffling, you should supply the parameter explicitly as well

Reply:
I'm also testing that the default argument is indeed shuffle, which we wouldn't notice if changed if we explicitly test positive.